paimon-web-server/src/main/java/org/apache/paimon/web/server/controller/SysRoleController.java

```diff
@@ -122,37 +124,37 @@ public R<List<SysRole>> all() {
     /** Query the list of assigned user roles. */
     @SaCheckPermission("system:role:list")
     @GetMapping("/authUser/allocatedList")
-    public PageR<User> allocatedList(User user) {
+    public PageR<User> allocatedList(@RequestBody RoleWithUserDTO roleWithUserDTO) {
         IPage<SysRole> page = PageSupport.startPage();
-        List<User> list = userService.selectAllocatedList(user);
+        List<User> list = userService.selectAllocatedList(roleWithUserDTO);
         return PageR.<User>builder().success(true).total(page.getTotal()).data(list).build();
     }
 
     /** Query the list of unassigned user roles. */
     @SaCheckPermission("system:role:list")
     @GetMapping("/authUser/unallocatedList")
-    public PageR<User> unallocatedList(User user) {
+    public PageR<User> unallocatedList(@RequestBody RoleWithUserDTO roleWithUserDTO) {
         IPage<SysRole> page = PageSupport.startPage();
-        List<User> list = userService.selectUnallocatedList(user);
+        List<User> list = userService.selectUnallocatedList(roleWithUserDTO);
         return PageR.<User>builder().success(true).total(page.getTotal()).data(list).build();
     }
 
     /** Unauthorize User. */
-    @SaCheckPermission("system:role:edit")
+    @SaCheckPermission("system:role:update")
     @PutMapping("/authUser/cancel")
     public R<Void> cancelAuthUser(@RequestBody UserRole userRole) {
```

Comment:
Could this interface merge with `cancelAuthUserAll`?

Reply:
cancelAuthUser used to delete one item by id of the user_role, cancelAuthUserAll used to batch delete by role_id with relation of userIds, I will merge them into one interface